app/views/carts/_cart.html.erb

```diff
@@ -120,9 +120,50 @@
         No comments have been added yet
       </p>
       <% end %>
-    <%- end %>
+    </div>
+
+    <div id="files">
+      <h3>Attachments to this proposal</h3>
+      <%= form_for [cart, Attachment.new] do |f| %>
+        <div class="line-item">
+          <div class="row">
+            <%= f.file_field :file %>
+          </div>
+        </div>
+        <div class='row text-area-info-container'>
+          <div class='col-xs-7 col-sm-6 text-area-info-web'>
+            <p>
+              Attaching a file will associate it with this proposal
+            </p>
+          </div>
+          <p class='col-xs-5 col-sm-6 text-area-button'>
+            <%= submit_tag "Attach a File", id: :add_a_file %>
+          </p>
+        </div>
+      <%- end %>
+      <% cart.proposal.attachments.each do |attachment| %>
+        <div class="line-item">
+          <div class="row">
+            <p class="col-sm-6 col-xs-12">
+              <a href="<%= attachment.file.expiring_url(10*60) %>"><%= attachment.file_file_name %></a>
```

Comment:
What is the magic number here? Maybe move to a helper or decorator or something?